Packs/Core/ReleaseNotes/3_0_86.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Updated the error message on alert_id input when core-get-cloud-original-alerts fails when debugging playbooks.
+
+##### Indicators detection
+
+- Updated the error message on alert_id input when core-get-cloud-original-alerts fails when debugging playbooks.
```

Comment:
Updated the error message on the `alert_id` input when the `core-get-cloud-original-alerts` command fails while using the playbook debugger.